tf2onnx/onnx_opset/nn.py

```diff
@@ -736,9 +736,12 @@ def version_11(cls, ctx, node, **kwargs):
             const_empty_float.output[0],
             concat_shape.output[0]
         ]
+        transformation_mode = "asymmetric"
+        if "half_pixel_centers" in node.attr and node.attr["half_pixel_centers"].i:
+            transformation_mode = "half_pixel"
```

Comment:
Thanks for the contribution. Seems like this should address issue: https://github.com/onnx/tensorflow-onnx/issues/914 as well.

Is that assumption correct?

Reply:
@jignparm That's right. This should fix the issue.

Reply:
Thanks for the contribution and the unit test! LGTM.